sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala

```diff
@@ -310,17 +310,13 @@ abstract class Expression extends TreeNode[Expression] {
   /**
    * Returns true when two expressions will always compute the same result, even if they differ
    * cosmetically (i.e. capitalization of names in attributes may be different).
-   *
-   * See [[Canonicalize]] for more details.
```

Comment:
You would need to fix:

```
sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala:   * See [[Canonicalize]] for more details.
sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala:   * See [[Canonicalize]] for more details.
sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ExpressionSet.scala:  /** Constructs a new [[ExpressionSet]] by applying [[Canonicalize]] to `expressions`. */
sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ExpressionSet.scala: * See [[Canonicalize]] for more details.
sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ExpressionSet.scala:   * Returns a string containing both the post [[Canonicalize]] expressions and the original
sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/QueryPlanConstraints.scala:   * See [[Canonicalize]] for more details.
```
too. While we're here, can you supplement the comments (e.g., where to look) instead of just removing it out?

Reply:
Maybe can`Expression.canonicalized` to replace this `Canonicalize`